src/feedback/feedback.tpl

```diff
@@ -1,7 +1,9 @@
-<div id="{{id}}" class="feedback feedback-{{level}} {{#if popup}}popup{{/if}}">
+<div id="{{id}}" class="feedback feedback-{{level}} {{#if popup}}popup{{/if}}"
+     role="dialog" aria-describedby="{{dompurify msg}}" >
```

Comment:
Aria-describedby takes the id of the node, take a look: https://www.w3.org/TR/wai-aria-1.1/#aria-describedby